service.c

```diff
@@ -83,11 +84,12 @@ VOID ServiceStart(DWORD dwArgc, LPTSTR *lpszArgv)
 	{
 		CloseHandle(hServerStopEvent);
 	}
-
 	if(hEvents[1])
 	{
 		CloseHandle(hEvents[1]);
 	}
+    ReportStatusToSCMgr(SERVICE_STOPPED, NO_ERROR, 0);
+    exit(0);

```

Comment:
`ReportStatusToSCMgr(SERVICE_STOPPED, ...);` is called in `service_main`, why call it here? And why is `exit(0);` necessary?